.ci/msys_test.bat

```diff
@@ -0,0 +1,7 @@
+:: FIXME(equalsraf) currently not all tests work on Windows
+:: for now just run what we can
+mingw32-make functionaltest TEST_FILE=test\functional\eval\printf_spec.lua VERBOSE=1 || goto :error
```

Comment:
> I'm trying to get this to work but TEST_TAG=windows doesn't seem to have any effect (no tests are run). I assumed the only thing I had to do was add #windows in describe but maybe I was wrong.

That's the way it supposed to be supposed to be (and it works for me locally on linux). 

If we expect more than say half of the test files to work on windows soon, it might be better to blacklist the non-working with `pending()`. This will mean we gradually _remove_ blacklists as more and more tests work on windows, which might feel better as the tests look cleaner as windows coverage improves.